lutron-aurora/src/init.lua

```diff
@@ -0,0 +1,105 @@
+local zcl_clusters = require "st.zigbee.zcl.clusters"
+local Level = zcl_clusters.Level
+local ZigbeeDriver = require "st.zigbee"
+local capabilities = require "st.capabilities"
+local device_management = require "st.zigbee.device_management"
+local utils = require 'st.utils'
+
+local PowerConfiguration = zcl_clusters.PowerConfiguration
+local Groups = zcl_clusters.Groups
+
+local LAST_LEVEL_EVENT = "LAST_LEVEL_EVENT"
+local DEVICE_GROUP = "DEVICE_GROUP"
+--server: Basic, PowerConfiguration, Identify, TouchlinkCommissioning, 0xFC00 
+--client: Identify, Groups, OnOff, Level, OTAUpgrade, TouchlinkCommissioning
+
+local do_configure = function(self, device)
+  device:send(device_management.build_bind_request(device, PowerConfiguration.ID, self.environment_info.hub_zigbee_eui))
+  device:send(device_management.build_bind_request(device, Level.ID, self.environment_info.hub_zigbee_eui))
+  device:send(PowerConfiguration.attributes.BatteryPercentageRemaining:configure_reporting(device, 30, 21600, 1))
+end
+
+local function added_handler(self, device)
+  for _, component in pairs(device.profile.components) do
+    device:emit_component_event(component, capabilities.button.numberOfButtons({ value = 1 }))
+    device:emit_event(capabilities.button.supportedButtonValues({"pushed"}, {visibility = { displayed = false }}))
+  end
+  device:send(PowerConfiguration.attributes.BatteryPercentageRemaining:read(device))
+  device:send(PowerConfiguration.attributes.BatteryPercentageRemaining:read(device))
```

Comment:
Is this duplicate needed?

Reply:
nope, I was probably trying to copy that to send a different read message but realized there isn't really anything else to read on this device